test/colors-spec.js

```diff
@@ -0,0 +1,138 @@
+'use strict'
+
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+
+const chai = require('chai')
+const Jimp = require('jimp')
+const tinyColor = require('tinycolor2')
+
+chai.should()
+const expect = chai.expect
+
+const Colors = require('../lib/colors.js')
+
+describe('colors', function () {
+  let testDir
+  const slugsAndIconPaths = []
+
+  before(async function () {
+    console.info = () => {}
+    console.error = () => {}
```

Comment:
Is the idea here that logging should be visible when the build is run but not the tests? I get that, but I think `console.error` output could be helpful for diagnosis when this test fails.

Reply:
Also this means that any legit console.errors will be sent to the void

Reply:
:+1: